projects/demo/src/modules/components/color-picker/examples/3/index.ts

```diff
@@ -0,0 +1,24 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '@demo/emulate/change-detection';
+import {encapsulation} from '@demo/emulate/encapsulation';
+import {BehaviorSubject} from 'rxjs';
+
+@Component({
+    selector: `tui-color-picker-example-3`,
+    templateUrl: `./index.html`,
+    changeDetection,
+    encapsulation,
+})
+export class TuiColorPickerExample3 {
+    colors = new Map([
+        [`red`, `var(--tui-error-fill)`],
+        [`green`, `var(--tui-success-fill)`],
+        [`blue`, `var(--tui-info-fill)`],
+    ]);
+
+    color$ = new BehaviorSubject(`var(--tui-success-fill)`);
```

Comment:
Unnecessary to have it in an `Observable`, it can be a plain value.

Reply:
There is some delay when updating color without `Observable` (until hint appears) I don't want to manually trigger change detection

Reply:
That's weird but ok :)